etna/articles/models.py

```diff
@@ -272,6 +272,16 @@ class RecordArticlePage(TopicalPageMixin, ContentWarningMixin, BasePageWithIntro
     parent_page_types = ["collections.ExplorerIndexPage"]
     subpage_types = []
 
+    intro_image = models.ForeignKey(
+        get_image_model_string(),
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text=_("Square, rotated image to display in the page introduction"),
```

Comment:
Question: Does this help text make sense? Personally I try not to be hyper-specific as these things change over time

Reply:
Hey Will, I agree. We shouldn't be too specific - however, help text is changeable and hopefully the designs don't change from this. The editors had also asked for this (or something similar) so if it's okay, could we keep this? I am open to any other suggestions though :)